api_reference/source/conf.py

```diff
@@ -154,11 +73,37 @@ def setup(app):
 # specifying the natural language populates some key tags
 language = "en"
 
+# -- Versioning --------------------------------------------------------------
```

Comment:
Pydata theme versioning dropdown and selection logic to make it easier to view/select doc versions